egs/librispeech/asr/simple_v1/local/make_kn_lm.py

```diff
@@ -0,0 +1,377 @@
+#!/usr/bin/env python3
+
+# Copyright 2016  Johns Hopkins University (Author: Daniel Povey)
```

Comment:
This file is a verbatim-copy of https://github.com/kaldi-asr/kaldi/blob/master/egs/wsj/s5/utils/lang/make_kn_lm.py